cdap-data-fabric/src/main/java/io/cdap/cdap/data2/metadata/lineage/field/FieldLineageInfo.java

```diff
@@ -391,6 +408,11 @@ private void computeIncomingSummaryHelper(EndPointField field, Operation current
         outgoingEndPointFields.add(entry.getKey());
       }
     }
+    for (String field : droppedFields) {
+      ReadOperation read = fieldSources.get(field);
+      EndPointField endPointField = new EndPointField(read.getSource(), field);
+      outgoingSummary.put(endPointField, Sets.newHashSet((EndPointField) null));
```

Comment:
Don't put a null in a set, Construct an `EndPointField` which has null for each field, i.e, `new EndpointField(Endpoint.of(null), null)`, you can construct this static variable in `EndPointField` class and add a comment saying why this is needed.